src/tac2core.ml

```diff
@@ -945,8 +945,6 @@ let () =
     let ist = { env_ist = Id.Map.empty } in
     let lfun = Tac2interp.set_env ist Id.Map.empty in
     let ist = Ltac_plugin.Tacinterp.default_ist () in
-    (** FUCK YOU API *)
-    let ist = { ist with API.Geninterp.lfun = (Obj.magic lfun) } in
```

Comment:
Slight overlook here: you should still set the lfun field, just removing the `Obj.magic` call.

Reply:
thanks